site/en/docs/web-platform/compute-pressure/index.md

```diff
@@ -1,264 +1,221 @@
-information.
-
-[Compute Pressure](https://github.com/oyiptong/compute-pressure/) is a new,
-experimental API allows web applications to gain insight into the CPU
-utilization of the browser's device. The API provides a callback
-that receives periodic updates, allowing developers to observe changes in CPU
-utilization over time. A web app that uses this API can enable or
-disable heavyweight features in response to these
-observations. It's a feedback mechanism that unlocks new doors for adaptive
-behavior, which can in turn lead to smoother, cooler experiences across a range
-of hardware devices.
-
-## API status
-
-<div>
-
-| Step                                     | Status                                                                       |
-| ---------------------------------------- | ---------------------------------------------------------------------------- |
-| 1. Create explainer                      | [Complete](https://github.com/oyiptong/compute-pressure/blob/main/README.md) |
-| 2. Create initial draft of specification | [In Progress](https://github.com/WICG/compute-pressure/)                     |
-| 3. Gather feedback and iterate on design   | In progress                                                                  |
-| 4. Origin trial                          | Complete                                                                     |
-| 5. Launch                                | Not started                                                                  |
+  The Compute Pressure API offers high-level states that represent the pressure on the system.
+  It allows the implementation to use the right underlying hardware metrics to ensure that users
+  can take advantage of all the processing power available to them as long as the system is not
+  under unmanageable stress.
+
+## Current status
+
+<div class="table-wrapper scrollbar">
+
+| Step                                     | Status                              |
+| ---------------------------------------- | ----------------------------------  |
+| 1. Create explainer                      | [Complete][explainer]               |
+| 2. Create initial draft of specification | [Complete][specs]                   |
+| 3. Gather feedback & iterate on design   | [In progress](#feedback)            |
+| 4. Origin trial                          | Not Started                         |
+| 5. Launch                                | Not started                         |
 
 </div>
 
-## API Availability
+### Try out the Compute Pressure API
+
+To experiment with the Compute Pressure API locally, read this [page][how-to].
+
+## Interfaces
+
+Pressure Observer API can be run in the following contexts:
+- Window or main thread
+- Dedicated Worker
+- Shared Worker
+
+The Compute Pressure API defines two new interfaces.
+
+`PressureObserver`: An object to observe the compute pressure of any number of sources at a
```

Comment:
`source` is used a lot and should be defined early in the doc. Can we explain first iteration in Chromium only exposes CPU source but others may come later such as...